MyLibrary/MyLibraryApp.DataAccess/Entities/Status.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyLibraryApp.DataAccess.Entities
+{
+    public class Status
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public int NoOfDays { get; set; }
```

Comment:
I think it would be easier for you if you keep the NoOfDays in the Order entity and here you should have only a few entries in the Status table with the status name.